.github/workflows/python-app.yml

```diff
@@ -35,8 +35,12 @@ jobs:
         python -m pip install --upgrade pip
         pip install -r requirements.txt
         pip install -r requirements-dev.txt
+        pip install pip-audit==2.2.0
```

Comment:
I think we should add this to requirements-dev.txt

Reply:
I agree that this should be changed

Reply:
What is the file requirements-dev.txt used for?

Reply:
It's used to put all the dependencies that a developer needs, but aren't needed to run to app in production. It allows the users to install the dev dependencies and run locally the kinds of checks that happen during the workflows. That way, a dev can perform the checks locally and fix any problems before pushing to the server.